code/modules/clothing/under/accessories/armband.dm

```diff
@@ -5,6 +5,18 @@
 	item_color = "red"
 	slot = "armband"
 
+/obj/item/clothing/accessory/armband/sec
+	name = "security armband"
+	desc = "An armband, worn by the crew to display which department they're assigned to. This one is white and red."
+	icon_state = "whitered"
+	item_color = "whitered"
+
+/obj/item/clothing/accessory/armband/yb
+	name = "blue-yellow armband"
+	desc = "A fancy blue and yellow armband!"
+	icon_state = "whitered"
```

Comment:
🤔

Reply:
I HAVE NO IDEA WHAT YOU ARE TALKING ABOUT EVERYTHING LOOKS NORMAL

Reply:
Okay, NOW I see it